lib/tasks/evm_settings.rake

```diff
@@ -81,16 +81,22 @@ module EvmSettings
   private_class_method :keyval_to_hash
 
   def self.str_to_value(value)
-    return true  if value =~ /true/i
-    return false if value =~ /false/i
-    return nil   if value =~ /nil/
+    return value unless value.kind_of?(String)
+
+    dvalue = value.downcasex
+    return true  if dvalue.include?("true")
+    return false if dvalue.include?("false")
+    return nil   if value.include?("nil")
```

Comment:
```suggestion
    return nil   if dvalue.include?("nil")
```

Reply:
I think this was intentional, but it's more consistent this way.

Reply:
the previous code was `/nil/` and not `/nil/i`
But I agree, this probably looks better. will make that change